### PR TITLE
feat: add mysql server

### PR DESCRIPTION
**Pull request by olipayne**
## Description
Added mysql server 

## Server Details
- Server: mysql-server

## Motivation and Context
Wanted to connect to a local MySQL database

## How Has This Been Tested?
Claude Desktop, working well

## Breaking Changes
No

## Types of changes
- [x] New MCP Server
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update

## Checklist
- [x] I have read the [MCP Protocol Documentation](https://modelcontextprotocol.io)
- [x] My server follows MCP security best practices
- [x] I have updated the server's README accordingly
- [x] I have tested this with an LLM client
- [x] My code follows the repository's style guidelines
- [x] New and existing tests pass locally
- [x] I have added appropriate error handling
- [x] I have documented all environment variables and configuration options

## Additional context
None

### CODE REVIEW

File: src/mysql/index.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    })),
+  };
+});
+
+server.setRequestHandler(ReadResourceRequestSchema, async (request: ReadResourceRequest) => {
+  const resourceUrl = new URL(request.params.uri);
+
+  const pathComponents = resourceUrl.pathname.split("/");
+  const schema = pathComponents.pop();
+  const tableName = pathComponents.pop();
```
Comment (dsp-ant) [inline src/mysql/index.ts:65]: We should probably be a bit more strict about the exact format here, e.g. test if there are more than two path components

File: src/mysql/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@modelcontextprotocol/server-mysql",
+  "version": "0.1.0",
```
Comment (dsp-ant) [inline src/mysql/package.json:3]: Since we are currently releasing all servers at the same time, they should all have matching versions.

**Review by dsp-ant — CHANGES_REQUESTED** [review]
Thank you! Some minor nits, but overall looking good. Really appreciated

File: src/mysql/index.ts [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+      await connection.beginTransaction();
+      
+      const [rows] = await connection.query(sql);
+      
+      return {
+        content: [{ type: "text", text: JSON.stringify(rows, null, 2) }],
+        isError: false,
+      };
+    } catch (error) {
+      throw error;
```
Comment (anaisbetts) [inline src/mysql/index.ts:121]: Why rethrow here?

**Pull request closed without merging**